src/pages/blog/recent-featured-posts/_style.js

```diff
@@ -0,0 +1,283 @@
+import styled from 'styled-components'
+import { Container, Flex } from 'components/containers'
+import { Text, Tabs } from 'components/elements'
+import { LocalizedLink } from 'components/localization'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+
+export const StyledContainer = styled(Flex)`
+    width: 90%;
+    max-width: 1200px;
+
+    @media ${device.desktopL} {
+        max-width: 1600px;
+    }
+    @media ${device.laptopM} {
+        width: 100%;
+        padding: 0 16px;
+    }
+`
+
+export const StyledTabs = styled(Tabs)`
+    width: unset;
+`
+
+export const ArticleContentWrapper = styled(Container)`
+    margin-top: 40px;
+    color: var(--color-white);
+    @media ${device.laptopM} {
+        flex-flow: column;
+    }
+`
+
+export const LeftContent = styled(Flex)`
+    display: flex;
+    justify-content: start;
+    margin-right: 10px;
+    @media ${device.laptopM} {
+        margin-right: 0;
+        margin-bottom: 20px;
+    }
+`
+
+export const RightContent = styled(Flex)`
+    display: flex;
+    justify-content: start;
+    @media ${device.laptopM} {
+        flex-flow: column;
+    }
+`
+
+export const MainArticle = styled(Flex)`
+    /* prettier-ignore */
+    background: ${(props) =>
+        props.image
+            ? `linear-gradient(251.14deg,rgba(14, 14, 14, 0.5632) 29.18%,rgba(7, 6, 6, 0.88) 85.14%),url(${props.image}) no-repeat top left`
+            : 'linear-gradient(251.14deg, rgba(14, 14, 14, 0.5632) 29.18%, rgba(7, 6, 6, 0.88) 85.14%)'};
+    position: relative;
+    background-size: cover;
+    min-height: 464px;
+    width: 792px;
+    max-height: 300px;
+    max-width: 1440px;
+    align-items: flex-end;
+    padding-bottom: 80px;
+    cursor: pointer;
+    transition: transform 0.3s;
+
+    &:hover {
+        transform: scale(1.02);
+    }
```

Comment:
Just personal preference, but might look nicer if we just scale the image inside the div instead of the entire container.

Reply:
I think we could leave this for future improvement? maybe the qa or the UX might have a different idea later on